### PR TITLE
1218 Thorchain Trade assets

### PR DESCRIPTION
**Pull request by 0xp3gasus**

### CODE REVIEW

File: packages/xchain-aggregator/src/types.ts [hunk 1]
```diff
@@ -14,9 +23,9 @@ type TxSubmitted = {
  * Fees
  */
 type Fees = {
-  asset: Asset | TokenAsset | SynthAsset // The asset for which fees are calculated
-  affiliateFee: CryptoAmount<Asset | TokenAsset | SynthAsset> // The affiliate fee amount
-  outboundFee: CryptoAmount<Asset | TokenAsset | SynthAsset> // The outbound fee amount
+  asset: Asset | TokenAsset | SynthAsset | TradeAsset // The asset for which fees are calculated
+  affiliateFee: CryptoAmount<Asset | TokenAsset | SynthAsset | TradeAsset> // The affiliate fee amount
+  outboundFee: CryptoAmount<Asset | TokenAsset | SynthAsset | TradeAsset> // The outbound fee amount
```
Comment (GrassRootsCrypto) [inline packages/xchain-aggregator/src/types.ts:28]: Outbound fee won't be a trade asset. Trade Assets live within THORChain, even with Order books.

Comment (stoicdev6) [reply packages/xchain-aggregator/src/types.ts:28]: there's any problem if i do this change?

**Review by Thorian1te — APPROVED** [review]
Had a Quick Look. 
Send it

File: packages/xchain-aggregator/src/types.ts (hunk 1 above)
Comment (0xp3gasus) [reply packages/xchain-aggregator/src/types.ts:28]: Looking at [documentation](https://thornode.ninerealms.com/thorchain/doc), in the QuoteFees type, the asset is the one used for all fees. If you are right, and outBoundFee is the native asset fee of the target fee. The bug is already there, we can create a new issue to fix

**Review by hippocampus-web3 — APPROVED** [review]

**Pull request merged**